specs/create_Edit_pool.adoc

```diff
@@ -0,0 +1,113 @@
+// vim: tw=79
+
+= Create and Edit Pool
+
+Write a functionality for creating and editing a Pool in Tendrl-Dashboard UI
+
+
+== Problem description
+
+For Creating the new Pool we need to Provide forms with different fields
+* Pool Name
+* Number of Pools to create
+* Type()
+* Replicas
+* Cluster it belongs
+* Journal configration
+* OSD's
+* PG Count
```

Comment:
User needs to use PG Calc tool and enter the value. tendrl is not going to provide an auto-calculated value.